notebooks/demo2/benchmarks.ipynb

```diff
@@ -0,0 +1,1612 @@
+{
```

Comment:
<p>Remove this cell</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/os-climate/aicoe-osc-demo/pull/194/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>